modules/telemetry/v1/data_block.py

```diff
@@ -18,14 +20,15 @@ class DataBlockSubtype(IntEnum):
     ANGULAR_VELOCITY = 0x05
     GNSS_LOCATION = 0x06
     GNSS_METADATA = 0x07
-    RESERVED = 0x3F
+    HUMIDITY = 0x08
+    RESERVED = 0xFF
```

Comment:
No need to add reserved (even though it was there before). In the spec it just shows that anything > 0x08 isn't used yet.

Reply:
Personally I agree with it being there for test cases/etc as a "default" value to set things to.